lib/pages/register/content/index.js

```diff
@@ -3,25 +3,25 @@ module.exports = {
   existingUser: 'Accept invitation',
   fields: {
     password: {
-      label: 'Select a password',
+      label: 'Enter a password',
       hint: 'Must be at least 8 characters'
     },
     confirm: {
       label: 'Confirm your password'
     }
   },
   account: {
-    setup: 'Your account will be created using {{email}}'
+    setup: 'Your account will be created using {{email}}.'
   },
-  acceptInvitation: 'Click the button below to accept this invitation and to link your profile to this establishment',
+  acceptInvitation: 'Create a password to accept this invitation and link your profile to this establishment.',
```

Comment:
I have a feeling that this same text is used for the case where a user is adding a second establishment, and so will already have a password set. Is that right @joefitter?

Reply:
this is specifically for the case where they already have a password set and are just linking to a new establishment